plugins/com.contrastsecurity.ide.eclipse.core.tests/src/com/contrastsecurity/ide/eclipse/core/SdkTest.java

```diff
@@ -0,0 +1,167 @@
+package com.contrastsecurity.ide.eclipse.ui.test;
```

Comment:
Update package name. Should now be **com.contrastsecurity.ide.eclipse.core**

Reply:
Done